scripts/validate-licenses.ts

```diff
@@ -29,6 +29,7 @@ const licensesWhitelist = [
   // Regular valid open source licenses supported by Google.
   'MIT',
   'ISC',
+  'Apache',
```

Comment:
That’s not a valid SPDX identifier.  There are three versions of the Apache license.

Can we fix the package instead?

Reply:
I'll try to do that, Probably they are all in the same mono repo